collection-scripts/targeted_logs

```diff
@@ -84,7 +85,7 @@ for nsvm in ${target_vms[@]}; do
 done
 
 # Collect CDI/importer pod logs for imported DVs
-for nsdv in ${target_dvs[@]}; do
+for nsdv in ${target_dvs[@]} ${target_pvcs[@]}; do
```

Comment:
ok, so I understand why you added PVCs here, but we're not going to have importer pods for those PVCs, right? I'm looking at the code below and see things like `$(oc get pods --no-headers -n $ns --selector cdi.kubevirt.io=importer -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep $dv | head -1)`, will we get something for the PVCs here at all?

Reply:
yes. dv name and pvc name are the same in the case we use dv. the vm contains only pvcs even when using dvs. we do use dvs in case such warm migrations / non local host destination. in that case, the pvc(==dv name) will bring up the right importer pod.

Reply:
ok, I didn't say it explicitly but I meant to flows in which we use volume populators and you replied on the other flows.
about the flows you had in mind - ok, you're right that when CDI is used it would be ok to use the PVC names but why having both methods then? can't we reply only on PVC names instead of DV names to get the importer pods?
and about flows in which volume populators are used, I'd prefer not to have logs like `Importer Pod for $nsdv doesn't exist, skipping.` when we don't expect to have importer pods..

Reply:
leaving the DVs for me is safety. if the vol-pop will be executed we will get the log you wrote, also for the opposite when passing on the migration and there is no vol-pop pod.

Reply:
> leaving the DVs for me is safety.

can you rephrase / elaborate?

> if the vol-pop will be executed we will get the log you wrote, also for the opposite when passing on the migration and there is no vol-pop pod.

ok, so should we treat both cases differently? that would also solve a syntactic issue with setting `nsdv` to a pvc that has not been originated from a data volume..

Reply:
> > leaving the DVs for me is safety.
> 
> can you rephrase / elaborate?
> 
mainly for supporting older versions.
but thinking on the way we go - we can drop DVs (unless we are going back to them?)
> > if the vol-pop will be executed we will get the log you wrote, also for the opposite when passing on the migration and there is no vol-pop pod.
> 
> ok, so should we treat both cases differently? that would also solve a syntactic issue with setting `nsdv` to a pvc that has not been originated from a data volume..

sure, if we are not going to drop DVs based on the above question

Reply:
we can't drop data volumes for warm migration and we now use them for OCP to OCP - so they're not going to disappear anytime soon